charts/core/templates/controller-secret.yaml

```diff
@@ -27,7 +27,7 @@ data:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: neuvector-internal-certs
```

Comment:
This is the crux of the issue. This value is not currently editable even when a different `secretname` is specified.